### PR TITLE
adding four new plays to udracor

### PR DESCRIPTION
**Pull request by DanilSko**

### CODE REVIEW

File: tei/kvitka-osnovjanenko-svatannja-na-goncharivci.xml [hunk 1]
```diff
@@ -0,0 +1,3509 @@
...
+            <speaker>Одарка.</speaker>
+            <p>Та так: Прибитий на цвiту.</p>
+          </sp>
+          <sp who="#kandzûba">
+            <speaker>Кандзюба.</speaker>
+            <lg>
+              <l>Тю-тю!Та ще фiть, фiть!</l>
+              <l>Ви дiло тут кажiть:</l>
+              <l>Чи вiддасте?</l> 
+              <stage>(с насмешкою)</stage>.
```
Comment (cmil) [inline tei/kvitka-osnovjanenko-svatannja-na-goncharivci.xml:420]: Same as above.

**Comment by DanilSko** [comment]
> all 4 files are missing a sourceDesc element

Added them here:  https://github.com/dracor-org/udracor/pull/7/commits/9002b7844453e3ff6b251a5668cee3799e97c82e

>Also, all TEI files in the repo are missing a publicationStmt

Added them to these fours plays here: https://github.com/dracor-org/udracor/pull/7/commits/702dbc8fdf79ba184a43a264850219df119353e2
Will add the rest publicationStmt-s in a separate PR

File: tei/shevchenko-nazar-stodolja.xml [hunk 2]
```diff
@@ -0,0 +1,2230 @@
...
+          </person>
+          <person xml:id="tretij">
+            <persName>Третий</persName>
+          </person>
+        </listPerson>
+      </particDesc>
+    </profileDesc>
+  </teiHeader>
+  <text>
+    <body level="0">
```
[the remark this answers is not in the document]
Comment (DanilSko) [reply tei/shevchenko-nazar-stodolja.xml]: Fixed here: https://github.com/dracor-org/udracor/pull/7/commits/6eabf52bdd444a27bcc287ab1ce27eb25ef72105

[the remark this answers is not in the document]
Comment (DanilSko) [reply tei/shevchenko-nazar-stodolja.xml]: Fixed here: https://github.com/dracor-org/udracor/pull/7/commits/6eabf52bdd444a27bcc287ab1ce27eb25ef72105

File: tei/kvitka-osnovjanenko-svatannja-na-goncharivci.xml [hunk 3]
```diff
@@ -0,0 +1,3509 @@
...
+              <l>От i треба добувати. </l>
+              <l>Та на вольну швидш чухрати.</l> 
+              <l>А де твоя гривня?</l>
+            </lg>
+          </sp>
+          <sp who="#odarka">
+            <speaker>Одарка</speaker>
+            <stage>(мужу)</stage>
+            <p>Вiдв'яжися; врагова дитина!</p> 
+            <stage>(Кандзюбе)</stage>.
```
[the remark this answers is not in the document]
Comment (DanilSko) [reply tei/kvitka-osnovjanenko-svatannja-na-goncharivci.xml]: https://github.com/dracor-org/udracor/pull/7/commits/1e898d5b9bf62d67941e5d182d5760b5ba6193da

File: tei/kvitka-osnovjanenko-svatannja-na-goncharivci.xml (hunk 1 above)
Comment (DanilSko) [reply tei/kvitka-osnovjanenko-svatannja-na-goncharivci.xml]: Fixed here: https://github.com/dracor-org/udracor/pull/7/commits/1e898d5b9bf62d67941e5d182d5760b5ba6193da

**Comment by DanilSko** [comment]
Ok, so I think the fixes are done @cmil 
> Would you perhaps find it helpful to add a validation workflow to udracor the same way I did for [epdracor](https://github.com/dracor-org/epdracor/actions)?

yes, I'd be happy!

**Comment by cmil** [comment]
> Ok, so I think the fixes are done @cmil

It looks good, the new files validate against TEI_all now. Thanks!

> > Would you perhaps find it helpful to add a validation workflow to udracor the same way I did for [epdracor](https://github.com/dracor-org/epdracor/actions)?
> 
> yes, I'd be happy!

I will set it up then.

**Comment by cmil** [comment]
One last thing before merging: let's not add the `tei/.DS_Store` to git. We could add this file to .gitignore, but for Mac users it's also a good idea to add it to a `.gitignore_global` in their home directory. Mine looks like this:

```
*~
.DS_Store
.vscode
```

**Comment by DanilSko** [comment]
> @DanilSko One last thing before merging: let's not add the `tei/.DS_Store` to git. 

Ok done here! https://github.com/dracor-org/udracor/pull/7/commits/aefff60a04e4661284ff6c3eb93cbe1c47c453ba

> for Mac users it's also a good idea to add it to a `.gitignore_global` in their home directory.

Done that too, thanks for the hint!

**Comment by DanilSko** [comment]
I see that TEI validation did not pass yet, but it seems to fail on all files, not only the ones i am adding. So I'd suggest i fix it in a next PR, is that ok?

**Comment by DanilSko** [comment]
Ok @cmil I think it validates against TEI schema now: https://github.com/dracor-org/udracor/actions/runs/3312912149/jobs/5470177743 
What do you think, shall we merge?

**Comment by cmil** [comment]
> Ok @cmil I think it validates against TEI schema now: https://github.com/dracor-org/udracor/actions/runs/3312912149/jobs/5470177743
> What do you think, shall we merge?

Yes, definitely. Great work!

**Pull request merged**